models/sql-utils.go

```diff
@@ -0,0 +1,36 @@
+package models
+
+import (
+	"fmt"
+	"strings"
+)
+
+// sanitizeOrderInput takes in the "order by" query, a validColums
+// string slice and returns a sanitized query
+//
+// it will allow to run order by query only on the columns that are present
+// in the validColumns string slice, if any other column is requested in the
+// query then it will be IGNORED and an empty query would be returned instead
+//
+// sanitizeOrderInput also expects the query to be no longer than two words, that is
+// the query may look like "updated_at DESC" or "name ASC"
+func sanitizeOrderInput(order string, validColumns []string) string {
+	parsedOrderStr := strings.Split(order, " ")
+	if len(parsedOrderStr) != 2 {
+		return ""
+	}
+
+	inputCol := parsedOrderStr[0]
+	typ := strings.ToLower(parsedOrderStr[1])
+	for _, col := range validColumns {
+		if col == inputCol {
+			if typ == "desc" {
+				return fmt.Sprintf("%s desc", col)
+			}
+
+			return fmt.Sprintf("%s asc", col)
+		}
+	}
+
+	return ""
```

Comment:
I wonder if we should be logging an error or if that leaves Meshery open to a potential overflow of logs.

Reply:
I am not sure if logging an error that an invalid "order by" query was passed will be helpful as if for some reason the query was actually malformed then it is probably an attack. However, there could be a case when a developer may have written a malformed query while developing, so for this use case adding a debug log in here maybe helpful in debugging.